website/src/content/docs/docs/migrate-swagger/01-get-started.md

```diff
@@ -9,43 +9,21 @@ We have created a swagger to TypeSpec conversion tool to help take on the bulk o
 
 **_Important!_** Because TypeSpec is more expressive than Swagger and with the help of evolving Azure libraries, this tool should only be used as an aid in the conversion/migration process, not as the sole tool to produce final version of TypeSpec specs without human inspection, correction and optimization.
 
-## Steps of running the tool
+## Steps of migration and comparison
 
-- Ensure [Node.js](https://nodejs.org/en/download/) 18.3 LTS or later is installed.
-- Install [`@azure-tools/typespec-client-generator-cli`](https://www.npmjs.com/package/@azure-tools/typespec-client-generator-cli):
-
-### Update existing swagger files
-
-- Run the tool to sort existing swagger so you can easily compare with TypeSpec generated swagger. Please note this functionality has been added in version 0.10.0. Please update to latest if you don't see this command option.
+### Prerequisite
 
+- Clone [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs).
+- Install dependencies:
```

Comment:
what are required dependencies to install?

Reply:
It's in next sentence. Run `npm install`